error.js

```diff
@@ -0,0 +1,15 @@
+exports.handlePsqlErrors = (err, req, res, next) => {
+  console.log(err);
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "invalid request" });
+  } else if (err.code === "23503") {
+    res.status(400).send({ msg: "invalid content" });
+  } else next(err);
+};
+
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status) {
+    console.log(err);
```

Comment:
console log! The only one that's OK to leave in is for a 500 server err, but we don't need to log custom errors if we're not debugging